### PR TITLE
Add XSL support

### PR DESCRIPTION
**Pull request by gucong3000**
https://github.com/Microsoft/vscode/blob/11b0f3a40be055e02b721abbb93cb0012818f99e/extensions/xml/package.json#L74-L78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Once a user follows [the stylelint startup guide](https://github.com/stylelint/s
 * TypeScript (`typescript`)
 * TypeScript React (`typescriptreact`)
 * [Vue (`vue`, `vue-html`)](https://marketplace.visualstudio.com/items?itemName=octref.vetur)
-* XML (`xml`)
```
Comment (shinnn) [inline README.md]: Include https://github.com/gucong3000/postcss-syntax/commit/5f2b1a92e800ee83eb32b8a060175f5c19b30c59 to the released version of postcss-syntax first, then we no longer need to remove this line.

Comment (shinnn) [reply README.md]: ⬆️

File: package.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
 		"onLanguage:typescriptreact",
 		"onLanguage:vue",
 		"onLanguage:vue-html",
-		"onLanguage:xml"
```
Comment (shinnn) [inline package.json]: Same as https://github.com/shinnn/vscode-stylelint/pull/214#discussion_r195948515

File: README.md [hunk 3]
```diff
@@ -48,6 +48,7 @@ Once a user follows [the stylelint startup guide](https://github.com/stylelint/s
 * TypeScript (`typescript`)
 * TypeScript React (`typescriptreact`)
 * [Vue (`vue`, `vue-html`)](https://marketplace.visualstudio.com/items?itemName=octref.vetur)
+* XSL (`xsl`)
```
Comment (shinnn) [inline README.md]: Keep A-Z order.

File: package.json (hunk 2 above)
Comment (gucong3000) [reply package.json]: Applying an XML file to stylelint is a futile.

Comment (gucong3000) [reply package.json]: Do we have any way to write CSS in XML files?

Comment (shinnn) [reply package.json]: `xml` language mode of Visual Studio Code is also applied to SVG files, which possibly include `<style>` tags in their contents.

**Pull request merged**